moveit_simple/test/kuka_kr210.cpp

```diff
@@ -647,15 +647,18 @@ TEST_F(UserRobotTest, copy_current_pose)
   // Before the first execution robot should be at position "home"
   before_execution_1 = robot->getJointState();
   EXPECT_NO_THROW(robot->execute(TRAJECTORY_NAME));
+  ros::Duration(0.5).sleep();
```

Comment:
Why is this needed?

Reply:
I think we had this discussion for the mh5 tests, added something similar [there](https://github.com/plusone-robotics/moveit_simple/blob/kinetic-devel/moveit_simple/test/motoman_mh5.cpp#L750). It was something about the industrial_robot_simlator was still at the end of the execution before the next getJointState() was called, causing the tests to sometimes fail or sometimes pass.